content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -496,3 +496,24 @@ Then, simply bind both `VirtualServices` to it like this:
 - `Gateway` configuration `gw` with host `*.test.com`, selector `istio: ingressgateway`, and TLS using gateway's mounted (wildcard) certificate
 - `VirtualService` configuration `vs1` with host `service1.test.com` and gateway `gw`
 - `VirtualService` configuration `vs2` with host `service2.test.com` and gateway `gw`
+
+### Configuring SNI routing when not sending SNI
+
+An HTTPS `Gateway` configuring the `hosts` field will perform an [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication) match on incoming requests.
+For example, the following configuration would only allow requests that match `*.example.com` in the SNI:
+
+{{< text yaml >}}
+servers:
+- port:
+    number: 443
+    name: https
+    protocol: HTTPS
+  hosts:
+  - "*.example.com"
+{{< /text >}}
+
+This may cause certain requests to fail.
+
+For example, if you do not have DNS set up and are instead directly setting the host header, such as `curl 1.2.3.4 -H "Host: app.example.com"`, no SNI will be set, causing the request to fail. Instead, you can set up DNS or use the `--resolve` flag of `curl`.
+
+Another common issue is load balancers in front of Istio. Most cloud load balancers will not forward the SNI, so if you are terminating TLS in your cloud load balancer you may need to either configure it to instead passthrough the TLS connection, or changes `hosts` to match `*`, which will disable the SNI matching.
```

Comment:
`changes hosts` -> `change hosts`

Also, you should be clear that you're referring to the `hosts` field in the `Gateway`. Maybe it could be restated `... or disable SNI matching in the Gateway by setting the hosts field to *`. Or maybe better yet, use a bullet list:

```
Another common issue is load balancers in front of Istio. Most cloud load
balancers will not forward the SNI, so if you are terminating TLS in your
cloud load balancer you may need to do one of the following:

- Configure it to instead passthrough the TLS connection
- Disable SNI matching in the `Gateway` by setting the hosts field to `*`
```

Reply:
@howardjohn how about specifically mentioning how health checks work general sni routing. I think we should use that example.